saleor_app_framework/app_framework/endpoints/manifest.py

```diff
@@ -0,0 +1,26 @@
+from fastapi import APIRouter, HTTPException
+from fastapi.requests import Request
+
+from ..core.manifest import get_app_manifest
+from ..schemas.manifest import Manifest
+
+router = APIRouter()
+
+
+@router.get(
+    "/manifest",
+    response_model=Manifest,
+    responses={502: {"description": "Manifest not found"}},
```

Comment:
Why 502 and not 404 for example?

Reply:
Hhm, I believe, I could just copy it from fast api. Now I am not sure what kind of the error should be placed here. I believe it should be something started from 5XX as client request is OK and, configuration of serwer is incorrect. 
Also, I believe that I can add some validation at the beggining of running the process. This should solve the issue. Thanks for catching this.

Reply:
@maarcingebala  I added fixme tag here.  I was planning to add a more user-friendly definition of the manifest.  I will back to this in future PRs.

Reply:
Sounds good 👍